reference/src/layout/structs-and-tuples.md

```diff
@@ -122,10 +124,32 @@ compiler will not reorder it, to allow for the possibility of
 unsizing. E.g., `struct Foo { x: u16, y: u32 }` and `struct Foo<T> {
 x: u16, y: T }` where `T = u32` are not guaranteed to be identical.
 
+#### Structs with no fields
+
+Structs with default layout and no fields are [1-ZST].
```

Comment:
Isn't this strictly redundant, subsumed by "Zero-sized structs"?

Reply:
Indeed.